documentation/samplefiles/readme.go.md

```diff
@@ -12,35 +12,15 @@ go:
 
 ``` yaml $(go) && $(multiapi)
 batch:
-  - tag: package-2019-12-01
-  - tag: package-2020-07-01-preview
-  # add every tag listed below
+  - tag: package--[[Version]]-[[ReleaseState]]
 ```
 
-### Tag: package-2019-12-01 and go
+### Tag: package-[[Version]]-[[ReleaseState]] and go
```

Comment:
```suggestion
### Tag: package-[[Version]] and go
```

Reply:
@ArcturusZhang you can view the placeholder meaning in https://github.com/JackTn/azure-rest-api-specs/blob/dev-tianenattestation-Microsoft.TianenAttestation-2020-09-27/specification/tianenattestation/resource-manager/readme.csharp.md . And i think it should contain Version and ReleaseState. Let me know if you have another question~